projects/core/package.json

```diff
@@ -3,13 +3,15 @@
     "version": "3.3.0",
     "description": "The main zero-dependency and framework-agnostic Maskito's package to create an input mask",
     "keywords": [
+        "input",
         "mask",
         "inputmask",
         "input-mask",
         "text-mask",
         "input-formatting",
         "javascript",
-        "typescript"
+        "typescript",
+        "react",
```

Comment:
I am not sure that we should add this keyword for framework-agnostic library `@maskito/core`.

`@maskito/react` already has this keyword – I think it is enough for any user search
https://github.com/taiga-family/maskito/blob/5640ed3a332b2816d8aad9fe083fb930769d7bef/projects/react/package.json#L2-L11